assets/timber.js.liquid

```diff
@@ -243,8 +245,11 @@ timber.productPage = function (options) {
       $comparePrice
         .html({{ 'products.product.compare_at' | t | json }} + ' ' + Shopify.formatMoney(variant.compare_at_price, moneyFormat))
         .show();
+      $comparePriceA11y.attr('aria-hidden', 'false');
+      $priceA11y.attr('aria-hidden', 'false');
```

Comment:
If it's never `aria-hidden=true` you don't have to set the attribute at all. Unset is the same as "false"